addons/core/translations/states/en-us.yaml

```diff
@@ -14,3 +14,6 @@ active-public: Public
 recording: Recording
 completed: Completed
 failed: Failed
+loading:
```

Comment:
I'm not sure what the best place was for this so I just added it to `states`, let me know if you have a better idea for this

Reply:
This seems like a good place for this IMO.